apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -1545,6 +1550,83 @@ settle_(TTL, MinDepth, #{fsm := FsmI, channel_id := ChannelId} = I, R, Debug,
     ct:log("died_normal detected from both", []),
     ok.
 
+client_reconnect_initiator(Cfg) ->
+    Debug = get_debug(Cfg),
```

Comment:
Wrap the test case inside a list comprehension with atoms [initiator, responder] as this is the only difference between the two test cases.

Reply:
I refactored, but not exactly like that.